app/policies/linked_profile_policy.rb

```diff
@@ -0,0 +1,11 @@
+class LinkedProfilePolicy < ApplicationPolicy
+  def show?
+    record.user == user || user.has_role?(:admin)
```

Comment:
isn't `is_admin?` only accessible in `create, update, destroy` because of the `alias_method`?
```
  def is_admin? # rubocop:disable Style/PredicateName
    user && user.has_role?(:admin, model_class)
  end
  alias_method :create?, :is_admin?
  alias_method :update?, :is_admin?
  alias_method :destroy?, :is_admin?
```